cmd/bosun/conf/conf.go

```diff
@@ -995,6 +997,8 @@ func (c *Conf) loadNotification(s *parse.SectionNode) {
 				c.error(err)
 			}
 			n.Body = tmpl
+		case "runOnActions":
```

Comment:
The use case here is that I want to email on actions, but maybe we don't want chat or some remote api or whatever to get those. This allows specific notifications to be exempt from action notifications.